pages/_app.jsx

```diff
@@ -2,25 +2,35 @@ import React from 'react'
 import NextApp from 'next/app'
 
 import Layout from '../components/layout'
+import { initializeData, GlobalProvider } from '../store'
+
+import '@oacore/design/lib/index.css'
 import './index.css'
 
 class App extends NextApp {
   static async getInitialProps({ Component, ctx }) {
     let pageProps = {}
+    const initialStoreData = initializeData()
```

Comment:
I feel this should be `await`

Reply:
Maybe but not at this time.

Reply:
It will make you more changes in the future. It's up to you do decide. I would prefer write everything properly form the start if I know about it know.

I am not a big fan of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) in such cases.